### DISCUSSION TITLE
Performance of some antennas

### DISCUSSION THREAD

**Discussion by mentanah** [egzumer/uv-k5-firmware-custom#435] [Show and tell]
today i had the opportunity to measure some antennas on a handheld spectrum analyzer from Rhode & Schwarz.
Here i would like to share the results in the pictures:

![antenna](https://github.com/egzumer/uv-k5-firmware-custom/assets/153547177/f92e56e8-8e02-40ae-9930-17e84d482098)
![SWR1](https://github.com/egzumer/uv-k5-firmware-custom/assets/153547177/5d577942-7ca5-473e-a09f-609cf427837d)
![SWR2](https://github.com/egzumer/uv-k5-firmware-custom/assets/153547177/a74bef1c-38c8-4b60-9b12-98c9014559da)
![SWR3](https://github.com/egzumer/uv-k5-firmware-custom/assets/153547177/ead5d270-e896-478c-9bea-98a554f279a6)
![SWR_expl](https://github.com/egzumer/uv-k5-firmware-custom/assets/153547177/481a5bd1-3092-4c96-9dfe-212f65945635)

**Comment by MrHoerby** [comment]
Die originale ist gar nicht so schlecht, nur ein wenig zu lang.

**Comment by OE1MWW** [comment]
Even using Rohde & Schwarz, the best of the best, all those measurements depend on the 'counterpoise' of such an antenna. 
Peak resonance may shift with the setup. They are all a kind of 1/4 radiator for 70cm and a radiator plus a loading coil for 2m, 
so they need metal for the other half side to complete the dipole or ground function.  Will be a major difference if the antenna 
sits on the UV-K5 or you have it connected via a coax to measure the SWR. Tried to put a choke at 1/4 distance of 2m and 70cm 
on the coax. OK, you may see a result, Next I used a small SWR bridge between UV-K5 and antenna - another result... 

German saying: "wer misst, der misst Mist. wer viel misst, der misst viel Mist" 
(whoever measures, measures crap. If you measure a lot, you measure a lot of crap) 

Experiment to do:
Use one of the shown antennas and squeeze a ~50cm wire to the SMA shield on the antenna connector of the UV-K5 and let 
the wire hang lose vertical down. Put your UV-K5 on exact the same place on a table, with or without the wire. Compare indicated 
signal info on the display of a (not so strong) repeater with or without the wire. You will be surprised ;-)

**Reply by mentanah** [reply]
That's right. This test is intended to do a relative comparison between the antennas. The setup is kept the same over measurements, with standing analyzer on grounded plane while I stepped back. 

Shure if you touch the sma ground nut with a fingertip, the low band drops about 20 MHz  and the high band about 50 MHz by increasing the SWR about 1 and 0.5 points.
